internal/convert.go

```diff
@@ -168,27 +168,52 @@ func (conv *Conv) SetDataMode() {
 	conv.mode = dataOnly
 }
 
+func dequeue(queue []string) []string {
```

Comment:
Suggest just putting this inline: the function is small and straightforward.

Reply:
yes, its just one line so we can put it inline.